daffodil-lib/src/main/scala/org/apache/daffodil/schema/annotation/props/PropertyScoping.scala

```diff
@@ -131,6 +144,21 @@ trait FindPropertyMixin extends PropTypes {
           lr
         }
       }
+    // if this looks like it could be an expression, issue a warning
+    // but only if expressionAllowed is false
+    if(!expressionAllowed) {
```

Comment:
Formatting: should be `if (!expressionAllowed) {` not `if(!expressionAllowed) {`